Library/Formula/fugu.rb

```diff
@@ -0,0 +1,87 @@
+require "language/go"
+
+class Fugu < Formula
+  desc "Docker command wrapper tool."
+  homepage "https://github.com/mattes/"
+  url "https://github.com/mattes/fugu/archive/v1.1.1.tar.gz"
+  sha256 "94ec61037c3afa1267ea990ffd03ba1d0d1628926b3fdde0133fece36fa81929"
+
+  depends_on "go" => :build
+  depends_on "docker" => :recommended
+
+  go_resource "gopkg.in/yaml.v2" do
+    url "https://gopkg.in/yaml.v2.git", :revision => "49c95bdc21843256fb6c4e0d370a05f24a0bf213"
+  end
+
+  go_resource "github.com/mattes/go-collect" do
+    url "https://github.com/mattes/go-collect.git", :revision => "7dc5ff4f9f3996f5ae618757e158fcbbaa8ab956"
+  end
+
+  go_resource "gopkg.in/mattes/go-expand-tilde.v1" do
+    url "https://gopkg.in/mattes/go-expand-tilde.v1.git", :revision => "cb884138e64c9a8bf5c7d6106d74b0fca082df0c"
+  end
+
+  go_resource "github.com/howeyc/gopass" do
+    url "https://github.com/howeyc/gopass.git", :revision => "10b54de414cc9693221d5ff2ae14fd2fbf1b0ac1"
+  end
+
+  go_resource "golang.org/x/crypto" do
+    url "https://github.com/golang/crypto.git", :revision => "7f27901cb9636a61b9b0b1bab2679b2a9a12eb38"
+  end
+
+  go_resource "github.com/docker/docker" do
```

Comment:
You already have `depends_on "docker"` above. Is it possible to avoid duplicating the dependency here?